repair_line_sequence/readme/CONTRIBUTORS.md

```
@@ -0,0 +1,3 @@
+Cetmix <cetmix.com>
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)